### DISCUSSION TITLE
在这个地方有问题

### DISCUSSION THREAD

**Discussion by unlomi** [natfrp/wiki#58] [水深火热]
映射隧道已经开好了，但是还是连不上服务器，我找了b站上所有的映射视频，没有跟我相似的情况，只在贴吧上找到了相同的情况，但是并没有发布解决方案。
![Screenshot_2022-05-17-15-48-10-608_com microsoft rdc android](https://user-images.githubusercontent.com/72535955/168760488-6fce545d-1c8d-44d6-bafc-4f0b53503e0c.jpg)
![Screenshot_2022-05-17-15-47-55-524_com mio launcher](https://user-images.githubusercontent.com/72535955/168760527-3395d43d-cb50-4b2a-bc37-acc46a67fc07.jpg)
![Screenshot_2022-05-17-15-47-52-435_com mio launcher](https://user-images.githubusercontent.com/72535955/168760539-3a187076-62da-450f-b6a0-fd7b70ec7b3c.jpg)

**Comment by hank9999** [comment]
枣庄节点需要先进行访问认证 再连接  

> 关于 枣庄机房 所有隧道强制启用访问验证的通知
> 出于合规需要，所有位于 枣庄多线 节点的隧道 自 2022-03-03 起 强制启用 [访问认证](https://doc.natfrp.com/#/bestpractice/security?id=frpc-%e8%ae%bf%e9%97%ae%e8%ae%a4%e8%af%81) 功能
> 对于目前 未设置访问密码 或使用的客户端为 启动器 v2.0.2.0 或 frpc v0.35.1-sakura-1 以下版本 或 frpc 不带 sakura 后缀的 枣庄隧道，您将需要在访问前通过管理面板隧道操作中的「授权访问」项目手动授权。
> 如果您使用的客户端是 启动器 v2.0.2.0 或 frpc v0.35.1-sakura-1 及以上版本，只需到管理面板填写访问密码后重启隧道即可通过文档中的方式进行验证，无需使用管理面板。
> ---引用自官网
